api/src/main/java/org/openmrs/module/aihdreports/metadata/Metadata.java

```diff
@@ -0,0 +1,335 @@
+
+package org.openmrs.module.aihdreports.metadata;
+
+import java.util.ArrayList;
+import java.util.HashMap;
+import java.util.List;
+import java.util.Map;
+
+import org.apache.commons.logging.Log;
+import org.apache.commons.logging.LogFactory;
+import org.openmrs.Concept;
+import org.openmrs.EncounterType;
+import org.openmrs.Form;
+import org.openmrs.Location;
+import org.openmrs.LocationTag;
+import org.openmrs.PatientIdentifierType;
+import org.openmrs.PersonAttributeType;
+import org.openmrs.RelationshipType;
+import org.openmrs.api.context.Context;
+import org.openmrs.module.reporting.common.ObjectUtil;
+
+public abstract class Metadata {
```

Comment:
I already have a class with this name here org.openmrs.module.aihdreports.reporting.metadata. Please consider renaming this or put what is useful in the former class